app/components/dashboard_course_component/dashboard_course_component.html.erb

```diff
@@ -30,10 +30,7 @@
     </div>
     <div class="dashboard-course-component__format">
       <div class="dashboard-course-component__format--icon">
-        <span class="govuk-body-s <%= course_icon_class(@achievement.activity) %>"></span>
-      </div>
-      <div class="dashboard-course-component__format--text">
```

Comment:
Do we no longer need the format--text class? If so, do we want the modifier --icon on the class, and could it just be a format div?

Reply:
Ah yes, I'll reduce this down to just a format div